style.css

```diff
@@ -660,17 +639,116 @@ li {
   letter-spacing: 0.03em;
   color: #6070ff;
   cursor: pointer;
+  padding: 2%;
 }
 
 .home_indicator {
   margin-top: 32px;
 }
 
+.popupOverlay {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  top: 0;
+  left: 0;
+  background-color: #c1c7d0;
+  backdrop-filter: blur(10px);
+  mix-blend-mode: multiply;
+  opacity: 0.9;
+  z-index: 2;
+  display: flex;
+  justify-content: center;
+  padding: 2em 0;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓